policies/security-policy.md

```diff
@@ -7,12 +7,13 @@ If you wish to report a possible security issue in OpenSSL please
 
 ## Issue triage
 
-Notifications are received by the OMC and OTC. We engage resources within
-OpenSSL to start the investigation and prioritisation. We may work in
-private with individuals who are not on the OMC or OTC as well as other
-organisations and our [employers](/community/thanks/)
-where we believe this can help with the issue investigation, resolution, or
-testing.
+Notifications are received by the OpenSSL Security Response Team (SRT)
+designated by the OpenSSL Foundation and the OpenSSL Corporation directors.
+
+We engage resources within OpenSSL organisations to start the investigation
```

Comment:
Leave it listed as OpenSSL Foundation and OpenSSL Corporation - we haven't used "organisations" before for it collectively.